documentation/source/specific.rst

```diff
@@ -14,6 +14,11 @@ Template Type Alias
 .. doxygentypedef:: IsFuzzy
    :path: ../../examples/specific/template_type_alias/xml
 
+.. doxygenfunction:: f0
+   :path: ../../examples/specific/class/xml
+.. doxygenfunction:: f0< std::string >
+   :path: ../../examples/specific/class/xml
+
```

Comment:
oh, it looks like I moved them. I don't remember why... I can move these back to the "Namespaced Function Examples" section.